[2024-07-22T02:31:12.552+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.sum_it scheduled__2022-03-26T00:00:00+00:00 [queued]>
[2024-07-22T02:31:12.599+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.sum_it scheduled__2022-03-26T00:00:00+00:00 [queued]>
[2024-07-22T02:31:12.606+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:12.612+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:31:12.622+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:31:12.653+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): sum_it> on 2022-03-26 00:00:00+00:00
[2024-07-22T02:31:12.668+0000] {standard_task_runner.py:55} INFO - Started process 718 to run task
[2024-07-22T02:31:12.678+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'sum_it', 'scheduled__2022-03-26T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpz_xdsuo5']
[2024-07-22T02:31:12.683+0000] {standard_task_runner.py:83} INFO - Job 125: Subtask sum_it
[2024-07-22T02:31:13.076+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 125 for task sum_it ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-03-26T00:00:00+00:00/task_id=sum_it/attempt=1.log'; 718)
[2024-07-22T02:31:13.161+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:31:13.303+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
